pnpm-lock.yaml

```diff
@@ -4480,7 +4484,7 @@ packages:
   /@types/bn.js@5.1.2:
     resolution: {integrity: sha512-dkpZu0szUtn9UXTmw+e0AJFd4D2XAxDnsCLdc05SfqpqzPEBft8eQr8uaFitfo/dUUOZERaLec2hHMG87A4Dxg==}
     dependencies:
-      '@types/node': 20.6.2
+      '@types/node': 16.18.12
```

Comment:
why is that happening? Is this on purpose?

Reply:
was just trying to figure this out - after doing some testing, it does not look like local node version (toggled using `nvm`) is the cause. i think it could be the installation of buffer that introduces this deep dependency change